src/z3c/autoinclude/utils.py

```diff
@@ -89,12 +92,20 @@ def distributionForDottedName(package_dottedname):
 
     if len(valid_dists_for_package) == 0:
         raise LookupError("No distributions found for package `%s`; are you sure it is importable?" % package_dottedname)
-    
+
     if len(valid_dists_for_package) > 1:
-        non_namespaced_dists = filter(lambda x: len(x[1]) is 0, valid_dists_for_package)
+        non_namespaced_dists = [x for x in valid_dists_for_package if len(x[1]) is 0]
```

Comment:
The horridness of that filter/lambda monstrocity originally obscured the `is 0` comparison from my sight, which can we *please* replace with `== 0`?  Or with `if not x[1]`?

Ideally I'd like to see `[(x, y) for x, y in valid_dists_for_package if not y]`, but with `x` and `y` replaced with meaningful names.